README.md

```diff
@@ -53,21 +53,22 @@ This project is called `ahab` as like the wild captain, it will kill the creatio
 
 At current time you have a few options:
 
-TBD
-
 ### Build from source
 
-TBD
-
 ```
 $ export GO111MODULE=on
 $ go test ./...
-$ go build
+$ go build -o ahab
 ```
 
 ### Download release binary
 
-TBD
+Each commit to master creates a new release binary, and if you'd like to skip building from source, you can download a binary similar to:
+
+```
+$ curl -O /path/where/you/want/ahab \
```

Comment:
FWIW I did this in Ubuntu, and I was getting all kinds of odd issues. If you want to give it a real life try, pop on over to: https://github.com/sonatype-nexus-community/example-ahab-repo/blob/master/Dockerfile and fix my stupidity :)

Reply:
O yeah looks like i fat fingered it. Should be lower case `-o`. So this

`curl -o ahab  https://github.com/sonatype-nexus-community/ahab/releases/download/v0.0.6/ahab-darwin.amd64-v0.0.6`

I can PR a change into that repo if you want and clean up the ahab binary there.

Reply:
Fixed up in 9caf7fb